tests/kafkatest/benchmarks/core/benchmark_test.py

```diff
@@ -152,8 +156,11 @@ def test_long_term_producer_throughput(self, compression_type="none", security_p
         self.logger.info("\n".join(summary))
         return data
 
+    @cluster(num_nodes=5)
     @parametrize(security_protocol='SSL', interbroker_security_protocol='PLAINTEXT')
-    @matrix(security_protocol=['PLAINTEXT', 'SSL', 'SASL_PLAINTEXT', 'SASL_SSL'], compression_type=["none", "snappy"])
+    @matrix(security_protocol=['PLAINTEXT', 'SSL'], compression_type=["none", "snappy"])
+    @cluster(num_nodes=6)
```

Comment:
An alternative is to tweak ducktape to allow a `num_nodes=function_of_test_parameters` in addition to an int. Then, the cluster size could be computed as a function of the injected arguments.